config/core/test-configs.yaml

```diff
@@ -1860,6 +1866,13 @@ test_configs:
     test_plans:
       - baseline
 
+  - device_type: at91sam9g20ek
+    test_plans:
+      - baseline
+      - baseline-nfs
```

Comment:
It failed to boot:
https://lava.sirena.org.uk/scheduler/job/36823

Reply:
Testsuite bug - I suspect the NFS root filesystem is a v7 one but this is an older machine that doesn't support that ISA.